client/settings/fraud-protection/components/fp-modal/index.js

```diff
@@ -18,6 +18,12 @@ export const HighFraudProtectionModal = ( {
 	setHighModalOpen,
 	storeCurrency,
 } ) => {
+	const { declineOnAVSFailure, declineOnCVCFailure } = window.wcpaySettings
```

Comment:
Added dynamics to AVS and CVC check texts, because we need to show/hide them according to the Stripe account settings for the merchant. If AVS checks are disabled, we need to hide them from both lists, for example.